### PR TITLE
Query tabular  issue 1428  

### PR DESCRIPTION
**Pull request by jj-umn**
query_tabular issue 1428 detect column names
    
    1. Allow selection of prefix when including column names in tabular
    output.
    2. Use values in first line of file for column names.  The names will
    be quoted if not a valid SQLite column name:  using double quotes,
    square brackets,  grave accents, or single quotes.  See:
    https://sqlite.org/lang_keywords.html

### CODE REVIEW

File: tools/query_tabular/test-data/psm_report.tsv [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (bgruening) [inline tools/query_tabular/test-data/psm_report.tsv]: @jj-umn why is the first line empty?

File: tools/query_tabular/query_tabular.xml [hunk 2]
```diff
@@ -406,6 +463,45 @@ Query Tabular
...
+  Specify Name for Table:
+
+    - By default tables will be named: t1, t2, t3, etc.
+ 
+    - **Specify Name for Table** 
+      You can provide your own name for a database table, the name should begin with a letter and only contain letters, digits, or underscores. 
+      The name should not be a SQLite key word: https://sqlite.org/lang_keywords.html
+
+
+  There are several way to name columns in a table:
```
Comment (bgruening) [inline tools/query_tabular/query_tabular.xml]: s/way/ways/

File: tools/query_tabular/test-data/psm_report.tsv (hunk 1 above)
Comment (jj-umn) [inline tools/query_tabular/test-data/psm_report.tsv]: Some versions of PeptideShaker printed a blank line, then the header line in the PSM report output.  That was one of the inspirations for regex line filtering.

File: tools/query_tabular/query_tabular.xml (hunk 2 above)
Comment (jj-umn) [inline tools/query_tabular/query_tabular.xml]: Will do.  Thanks.

File: tools/query_tabular/test-data/psm_report.tsv (hunk 1 above)
Comment (bgruening) [inline tools/query_tabular/test-data/psm_report.tsv]: Ok, thanks for the explanation and thanks again!

**Comment by bgruening** [comment]
Thanks JJ!

**Pull request merged**